posthog/tasks/exports/csv_exporter.py

```diff
@@ -94,6 +97,14 @@ def _convert_response_to_csv_data(data: Any) -> Generator[Any, None, None]:
             for row in results:
                 row_dict = {}
                 for idx, x in enumerate(row):
+                    if isinstance(x, dict):
+                        for key in filter(
+                            lambda y: y in RESULT_LIMIT_KEYS and len(x[y]) > RESULT_LIMIT_LENGTH, x.keys()
+                        ):
+                            total = len(x[key])
+                            x[key] = x[key][:RESULT_LIMIT_LENGTH]
+                            x[key].append(f"Note: {total} {key} in total")
```

Comment:
It's a bit weird that this is in the `distinct_ids.10` column

<img width="178" alt="Screenshot 2024-06-25 at 10 04 04" src="https://github.com/PostHog/posthog/assets/1851359/b585c495-d82c-442b-8218-6423c371e41c">

I'd prefer a `total_distinct_ids` column or similar

Reply:
Yeah that may be better for people importing this with scripts or something.